Library/Formula/ack.rb

```diff
@@ -1,4 +1,5 @@
 class Ack < Formula
+  desc "A search tool like grep, but optimized for programmers"
   homepage "http://beyondgrep.com/"
```

Comment:
I figured that you might check the name of the formula and the homepage and description yourself, but rely on `brew` to check the url and shasum for you.
What do other people think?